src/browser/main.js

```diff
@@ -14,7 +14,7 @@ import {browserHistory} from 'react-router';
 window.Promise = Bluebird;
 
 const app = document.getElementById('app');
-const initialState = window.__INITIAL_STATE__;
+const initialState = JSON.parse(app.getAttribute('data-initial-state'));
```

Comment:
Can't think about any particular reason, except aesthetic one) May be not a big deal